promises.cf.in

```diff
@@ -198,6 +199,19 @@ bundle common cfengine_stdlib
       # instead of lib/3.8
       "inputs" slist => { "$(sys.local_libdir)/stdlib.cf" };
 
+    !cfengine_3_7.windows.(cfengine_3_12|cfengine_3_13|cfengine_3_14|cfengine_3_15).!cfengine_3_12_0::
```

Comment:
do we need `!cfengine_3_7` here, @nickanderson ? I doubt any of `(cfengine_3_12|cfengine_3_13|cfengine_3_14|cfengine_3_15)` will be set there :)

Reply:
Ah, mmmm I guess you can remove that part. Nice find.